activesupport/test/core_ext/duration_test.rb

```diff
@@ -3,6 +3,7 @@
 require_relative "../abstract_unit"
 require "active_support/inflector"
 require "active_support/time"
+require "active_support/core_ext/numeric/bytes" # TODO: remove when Duration#method_missing is removed
```

Comment:
Much as I don't like having orphan `require`s around the place, the comment is rather unsightly in its own right. How bad would it be for us to push this inside the relevant test method? :thinking:

Reply:
Updated this to require in the test and restore methods after. It looks pretty ugly thought, it's certainly making me lean towards just not testing those methods explicitly.